site/cypress/e2e/Cypress-Gradeable/cyclic_grading_view.spec.js

```diff
@@ -0,0 +1,53 @@
+describe('Cyclic grading View Test', () => {
+    ['ta', 'instructor'].forEach((user) => {
+        it(`${user} cyclic grading view testing for Grading Homework`, () => {
+            cy.login(user);
+            cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'status']);
+            cy.get('[data-testid="Grading Index"]').should('exist').click();
+            if (user === 'ta') {
+                cy.get('[data-testid="popup-window"]').should('exist');
+                cy.get('[data-testid="close-button"]').should('exist');
+                cy.get('[data-testid="close-hidden-button"]').should('exist');
+                cy.get('[data-testid="agree-popup-btn"]').should('exist').click();
+            }
+            cy.get('[data-testid="view-sections"]').click();
+            cy.get('[data-testid="grade-button"]').eq(12).click();
+            cy.get('[data-testid="student-info-btn"]').click();
+            cy.get('.rubric-title').should('contain', 'Joe Student');
+            cy.get('#prev-student-navlink').click();
+            cy.get('.rubric-title').should('contain', 'John Smith');
+            cy.get('#grading-setting-btn').click(); // change
+            cy.get('#general-setting-list').find('.ta-grading-setting-option').first().select('Prev/Next Ungraded Student');
+            cy.get('.form-title > [data-testid="close-button"]').should('exist').eq(3).click();
```

Comment:
What is the point of this line? If you just want to close the modal, you can just use `cy.get([data-testid="close-button"]).click();`. Why should it equal 3?

Reply:
<img width="1352" alt="Screenshot 2024-05-24 at 5 10 09 PM" src="https://github.com/Submitty/Submitty/assets/119070053/5f21f762-98ed-47ed-903a-179ad7180303">

Removed the chaining , but look into the image there 4 more element with same data-testid
`cy.get('.form-title > [data-testid="close-button"]').should('exist').eq(3).click();`

Reply:
any comment on that ??
like its ok ?? to go with .eq()